pom.xml

```diff
@@ -636,6 +637,11 @@
                 <artifactId>pagehelper-spring-boot-starter</artifactId>
                 <version>${pagehelper.springboot.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.jsqlparser</groupId>
```

Comment:
Is any `NOTICE` or `LICENSE` file needed?

Reply:
fixed, thx

Reply:
No need, it supports Apache license V2.0.